drivers/led/is31fl319x.c

```diff
@@ -319,25 +318,28 @@ static DEVICE_API(led, is31fl3194_led_api) = {
 #define LED_CURRENT(led_node_id)						\
 	DT_PROP(led_node_id, current_limit),
 
-#define IS31FL3194_DEFINE(id)							\
+#define IS31FL319X_DEVICE(n, id)				\
+										\
+	DT_INST_FOREACH_CHILD(n, COLOR_MAPPING)					\
 										\
-	DT_INST_FOREACH_CHILD(id, COLOR_MAPPING)				\
+	static const struct led_info is31fl319##id##_leds_##n[] =		\
+		{ DT_INST_FOREACH_CHILD(n, LED_INFO) };			\
```

Comment:
issue here seems to be you are fixing in wrong commit, check first commit here, the line ending on lines 321 and 326 are wrong, they are wrongly added in the first commit then fixed in the second commit, can you fix them in the first commit?